src/cli/abrt-cli-core.c

```diff
@@ -148,13 +153,18 @@ void initialize_polkit_agent(void)
     }
 
     g_object_unref(subject);
+#else
+    printf("Polkit support is currently disabled\n");
```

Comment:
I agree with using log levels. I just didn't know how you preferred it to be done and now I do. I will push a new patch shortly. Is being at INFO level good enough to not spam the user? Or do you prefer that it only prints out the first time abrt-cli is run?